src/code42cli/click_ext/groups.py

```diff
@@ -0,0 +1,110 @@
+import difflib
```

Comment:
are all these `click_ext` files the same as they were before, just moved to another directory? What, if anything, has changed here?

Reply:
Yeah, just reorganized things a bit. Since I was adding the new `click.File` subclass (`JSONFileOrString`) and it didn't really fit anywhere, I decided to move all the re-usable custom click classes here.